test/WebJobs.Script.Tests/ExtensionBundle/ExtensionBundleManagerTests.cs

```diff
@@ -384,10 +385,36 @@ public async Task GetExtensionBundle_CannotReachZipEndpoint_ReturnsFalseAsync()
             Assert.Null(await manager.GetExtensionBundlePath(httpClient));
         }
 
+        [Theory]
+        [InlineData("[3.*, 4.0.0)", "3.19.0")]
+        [InlineData("[4.*, 5.0.0)", "4.2.0")]
+        [InlineData("[4.*, 5.0.0)", null)]
+        [InlineData("[3.*, 4.0.0)", null)]
+        public void LimitMaxVersion(string versionRange, string version)
```

Comment:
In this test, where are we setting the hosting config value for MaxBundle version? Are we are not exercising the new logic you added in the if blocks?

Reply:
We are exercising the new logic in the if blocks. That logic only applies if the hosting config for v3 and v4 versions are set. In this test case we are checking scenarios, when hosting config is set and when it is not. When hosting config is set the bundle version should be limited to max version mainly it can use a lower version but not higher. When hosting config is not set app can use any version within the version range.

Reply:
Okay, but I am still confused about where we are setting the hostingConfig value for the test? If we do not do that, the `configOption.MaximumBundleV3Version` will return `null` and the code inside if block won't be tested via this UT.

Reply:
Oh man, i missed that completely. That was a good catch. I have fixed the test now.